docs/userguide/canvas.rst

```diff
@@ -1122,3 +1122,53 @@ of one:
 
 This means that the first task will have a countdown of one second, the second
 task a countdown of two seconds, and so on.
+
+Stamping
+=======
+The goal of the Stamping API is to give an ability to mark signature and it components for a debug purposes. For example,
+
+.. code-block:: pycon
+
+    >>> sig1 = add.si(2, 2)
+    >>> sig1_res = sig1.freeze()
+    >>> g = group(sig1, add.si(3, 3))
+    >>> g.stamp(stamp='your_custom_stamp')
+    >>> res = g1.apply_async()
+    >>> res.get(timeout=TIMEOUT)
+    [4, 6]
+    >>> sig1_res._get_task_meta()['stamp']
+    ['your_custom_stamp']
```

Comment:
We should demonstrate the default behavior of stamping two values on the same key.